config/initializers/sidekiq.rb

```diff
@@ -2,4 +2,9 @@
   config.client_middleware do |chain|
     chain.add SidekiqLoggerMiddleware
   end
+
+  # See https://github.com/mhenrixon/sidekiq-unique-jobs/blob/921aaa4efc998b102790d1f4ecce2ebac609e464/README.md#add-the-middleware
+  config.client_middleware do |chain|
+    chain.add SidekiqUniqueJobs::Middleware::Client
```

Comment:
Looks like this block should be combined with the proceeding one.

Reply:
Thanks, done!